modules/renter/hostdb/persist.go

```diff
@@ -75,3 +71,27 @@ func (hdb *HostDB) load() error {
 	hdb.lastChange = data.LastChange
 	return nil
 }
+
+// threadedSaveLoop saves the hostdb to disk every 2 minutes, also saving when
+// given the shutdown signal.
+func (hdb *HostDB) threadedSaveLoop(shutChan chan struct{}) {
```

Comment:
I plan on repeating this pattern in other modules so we should figure out how we want to code it.

I think repeating this in other places will speed up IBD